### PR TITLE
Update the citation information

### LINKED ISSUE #160: Add citation information to readthedocs documentation

### PR DESCRIPTION
**Pull request by timtroendle**
* Mention citation on the landing page of readthedocs.
* Add citation information into About section of readthedocs.
* Add a CITATION.cff file to the repository.

Fixes #160  .

## Checklist

Any checks which are not relevant to the PR can be pre-checked by the PR creator. All others should be checked by the reviewer. You can add extra checklist items here if required by the PR.

- [x] CHANGELOG updated
- [x] Minimal workflow tests pass
- [x] Tests added to cover contribution
- [x] Documentation updated
- [x] Configuration schema updated

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,33 @@
+cff-version: 1.1.0
+message: If you use Euro-Calliope in academic publication, please cite the Joule research article listed in `references` below.
+date-released: 2020-07-17
+title: Euro-Calliope
+version: 1.1.0.dev
+doi: 10.5281/zenodo.3949793
+authors:
```
Comment (timtroendle) [inline CITATION.cff:7]: If we do this, I'll invite all contributors to add their names.

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,33 @@
+cff-version: 1.1.0
+message: If you use Euro-Calliope in academic publication, please cite the Joule research article listed in `references` below.
```
Comment (brynpickering) [inline CITATION.cff:2]:
```suggestion
message: If you use Euro-Calliope in an academic publication, please cite the Joule research article listed in `references` below.
```

File: CITATION.cff (hunk 1 above)
Comment (brynpickering) [reply CITATION.cff:7]: Makes sense to me.

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+date-released: 2020-07-17
+title: Euro-Calliope
+version: 1.1.0.dev
+doi: 10.5281/zenodo.3949793
```
Comment (brynpickering) [inline CITATION.cff:6]: Does this zenodo link match version `1.1.0.dev`?

File: docs/index.md [hunk 4]
```diff
@@ -33,3 +37,9 @@ By building the models yourself, you have many more options to configure, extend
 You should have some or expert knowledge in scientific workflows and execution environments, or be willing to learn about those.
 Building the models will require to download several gigabytes of raw data and a few hours of compute time on a regular personal computer.
 [Learn more about using your own builds.](./workflow/build.md)
+
```
Comment (brynpickering) [inline docs/index.md:40]: I would not have this and instead rely on it being in `citation.md`. You can always link to it in a sentence elsewhere in this file.

**Review by brynpickering — APPROVED** [review]
Minor comments; otherwise looks fine to me.

File: CITATION.cff (hunk 3 above)
Comment (timtroendle) [reply CITATION.cff:6]: No, it's the versionless DOI. One less url to update during the release process. It's going to be many even without this one.

File: docs/index.md (hunk 4 above)
Comment (timtroendle) [reply docs/index.md:40]: Ok. I removed it.

File: CITATION.cff (hunk 1 above)
Comment (timtroendle) [reply CITATION.cff:7]: :wave: @FebinKa , @suvayu, @sjpfenninger, @silas-schweizer : You have all contributed to v.1.1.0 of euro-calliope. If you would like to be mentioned as an author, please add your name here and open a pull request. You do not need an ORCID should you not have one.

**Pull request merged**